src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = () => {
+  return new Promise((resolve) => {
+    resolve('Promise was resolved!');
+  });
+};
+
+const promise2 = () => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(new Error('Promise was rejected!'));
+    }, 3000);
+  });
+};
+
+logo.addEventListener('click', () => {
+  promise1()
+    .then((message) => {
+      document.body.insertAdjacentHTML(
+        'beforebegin',
+        `<div class="message">${message}</div>`,
+      );
+    })
+    .catch((error) => {
+      document.body.insertAdjacentHTML(
+        'beforebegin',
+        `<div class="message error-message">Error: ${error.message}</div>`,
+      );
+    });
```

Comment:
Create a helper functions for succes and error messages.
For example you need to write helper function for error like this
```
const div = document.createElement('div');

div.classList.add('message', 'error-message');
div.textContent = message;

body.append(div);
```